docs/README.kr.md

```diff
@@ -11,17 +11,33 @@
 <br></br>
 [![Deploy to now](https://deploy.now.sh/static/button.svg)](https://deploy.now.sh/?repo=https://github.com/dawnlabs/carbon&env=TWITTER_CONSUMER_KEY&env=TWITTER_CONSUMER_SECRET&env=TWITTER_ACCESS_TOKEN_KEY&env=TWITTER_ACCESS_TOKEN_SECRET&env=LOGS_SECRET_PREFIX)
 
+
```

Comment:
Please don't add the "Translations" section here. I want to isolate the list to a single spot.

Reply:
@mfix22 Okay. Thank you for your quick response. 😄